strings/strings.c

```diff
@@ -18,13 +23,26 @@ int string_length(char *s)
     enough space for the reversed string. Don't forget to terminate 
     the reversed string with a null character. Return the rv array.
 */
-char *reverse_string(char *rv, char *s)
+char *reverse_string(char *rv, char *s) 
 {
-
+    // find length of string
+    int len = string_length(s);
+    // initialize pointer at end of string but before the null terminator
+    char *p = s + (len - 1);
+    // iterate through rv
+    for (int i = 0; i < len; i++) {
+        // sets value at rv[i] to current value of *p
+        *(rv + i) = *p;
```

Comment:
just pointing out that `*(rv + i)` also works as `rv[i]`, but of course use whichever your prefer